assets/scss/templates/_guidelines.scss

```
@@ -209,3 +209,24 @@ body {
     padding: 5px 0;
   }
 }
+
+
+/* tooltip */
+
+.wrapper-tooltip {
+  flex-wrap: wrap;
+  display: flex;
+  justify-content: space-between;
+  padding: 100px 10px;
+}
+
+.dp-box {
+  /* We can use the following parameters to customize them */
+  /* Mixin parameters $color - $width - $max-width - $min-height */
+
+  @include dp-box($dp-color-silver, 20%, 408px, 150px);
```

Maybe this way is better to explain what is each parameter:

```suggestion
  @include dp-box($color: $dp-color-silver, $width: 20%, $max-width: 408px, $min-height: 150px);
```

See https://sass-lang.com/documentation/at-rules/mixin#keyword-arguments

Does it work?

---

I turned this question around and found a way to do it. I had to modify it in the creation of @mixin and not in the call of @include, sorry for the delay, I think it's more readable, thanks